### PR TITLE
Finalized the project

### PR DESCRIPTION
**Pull request by muneebulrehman**
## Finalized the project with all the requirements
- Used redux for state management
- Used appropriate middlewares
- Used SpaceX api to fetch rockets and missions data
- Rockets as the main page
- Implemented Missions page and My Profile page
- Can reserve and cancel reservation of rockets
- Can join and leave a particular mission
- All the reserved rockets and joined missions are accessible from My Profile section
- Rockets page will load only once when application is loaded.

### CODE REVIEW

File: src/components/navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <nav>
+    <div className={style.logo}>
+      <img src={logo} alt="logo" />
+      <h2>Space Travelers&apos; Hub</h2>
+    </div>
+    <div className={style.list}>
+      <NavLink to="/">Rockets</NavLink>
+      <NavLink to="/missions">Missions</NavLink>
+      <div className={style.line} />
+      <NavLink to="/profile">My Profile</NavLink>
```
Comment (elyor-doniyorov) [inline src/components/navbar.jsx:16]: - [x] Please make sure that the active page is indicated with an underline so its look and feel be better.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @muneebulrehman and @souad988, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔️ There are no linter errors
✔️ [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start) exist on the package.json file
✔️ Proper routes and view components are created

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- The pull request link should be opened from the `dev `branch into the `main `branch and include all the changes. Kindly fix it! Read [this ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/code_review_flow_group_projects.md)article to get more information, please.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by muneebulrehman** [comment]
> Hi @muneebulrehman and @souad988,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Highlights
> ✔️ There are no linter errors ✔️ [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start) exist on the package.json file ✔️ Proper routes and view components are created
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> You can also consider:
> 
> * The pull request link should be opened from the `dev `branch into the `main `branch and include all the changes. Kindly fix it! Read [this ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/code_review_flow_group_projects.md)article to get more information, please.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@elyor-doniyorov According to the article you referenced, the first line says. Open a pull request from your feature branch to the development branch with changes that need a review. That is exactly what we did.

File: src/components/navbar.jsx (hunk 1 above)
Comment (muneebulrehman) [reply src/components/navbar.jsx:16]: Thanks for the suggestion. Have implemented the same. 🙂

**Review by Deepakdanger — APPROVED** [review]
**Hi @Team**, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by muneebulrehman** [comment]
Thanks a lot  😊✌️

**Pull request merged**